locust/main.py

```diff
@@ -344,11 +345,24 @@ def timelimit_stop():
         spawn_run_time_limit_greenlet()
 
     stats_printer_greenlet = None
+    input_listener_greenlet = None
     if not options.only_summary and (options.print_stats or (options.headless and not options.worker)):
         # spawn stats printing greenlet
         stats_printer_greenlet = gevent.spawn(stats_printer(runner.stats))
         stats_printer_greenlet.link_exception(greenlet_exception_handler)
 
+        input_listener_greenlet = gevent.spawn(
+            input_listener(
+                {
+                    "w": lambda: runner.spawn_users(1, runner.spawn_rate),
```

Comment:
no need to use runner.spawn_rate, just do it immediately (set it to maybe 100). If people want to spawn slowly then let them wait between keypresses :) (when spawning only one user it will not matter anyway)